locales/vi.schema.json

```diff
@@ -804,6 +804,13 @@
         },
         "margin_top": {
           "label": "Lề trên"
+        },
+        "header__8": {
+          "content": "Liên kết chính sách",
+          "info": "Để thêm chính sách cửa hàng, truy cập [cài đặt chính sách](/admin/settings/legal)."
+        },
+        "show_policy": {
+          "label": "Hiển thị liên kết chính sách"
         }
       }
```

Comment:
Do we need the circle if it goes to the 2nd line? (first circle)

https://screenshot.click/08-45-wd8so-jri89.png

cc @allylane